### DISCUSSION TITLE
Settings as part of the database

### DISCUSSION THREAD

**Discussion by seityaya** [keepassxreboot/keepassxc#8738] [Ideas]
Settings as part of the database

Enable database settings as part of the data. For example: password generator options, lock time, minimizing to tray and thus was possible. Ideally, all settings that can be made by the user and properties that depend on experience with the database.
Beyond appearance and language.

For example:
- when I open a base with personal pairs. I want the input time to be 20 seconds and the base to lock after 2 minutes.
- and when I work with the database of working passwords, the input time was 10 seconds, and the database was blocked after 10 seconds.

Sometimes I work with the settings on other machines, and I check the applications there is no time, what kind of delays are there, is there integration with browsers, and this was also required. I just don't want to worry about it.

**Comment by droidmonkey** [comment]
#2224

**Reply by seityaya** [reply]
The question is 5 years old. Will this functionality be implemented?

**Reply by droidmonkey** [reply]
Maybe, it's not a priority obviously
